packages/@sanity/desk-tool/src/pane/DocumentsPane.js

```diff
@@ -21,18 +22,37 @@ import Snackbar from 'part:@sanity/components/snackbar/default'
 
 const NOOP = () => {} // eslint-disable-line
 
-function readListLayoutSettings() {
-  return JSON.parse(window.localStorage.getItem('desk-tool.listlayout-settings') || '{}')
+const LOCALSTORAGE_KEY = 'desk-tool.documents-pane-settings'
+
+function readSettings() {
+  return JSON.parse(window.localStorage.getItem(LOCALSTORAGE_KEY) || '{}')
 }
 
-function writeListLayoutSettings(settings) {
-  window.localStorage.setItem('desk-tool.listlayout-settings', JSON.stringify(settings))
+function writeSettings(settings) {
+  window.localStorage.setItem(LOCALSTORAGE_KEY, JSON.stringify(settings))
 }
 
 function getDocumentKey(document) {
   return getPublishedId(document._id)
 }
 
+function toGradientOrderClause(orderBy) {
+  return Object.keys(orderBy).map(fieldName => `${fieldName} ${orderBy[fieldName]}`).join(', ')
+}
+
+const DEFAULT_SORT_OPTIONS = [
+  {
+    title: 'Last edited',
+    name: '__updatedAt',
```

Comment:
It's not clear to me why these are double-underscored - is this intentional? If so, maybe add a comment explaining why?

Reply:
You are right - The double underscores were there to prevent conflicts with possible user-defined sort options with the same name. But on second thought; being able to override the semantics of `updatedAt` is actually a nice feature that we could document.